src/processRequest.mjs

```diff
@@ -306,7 +305,6 @@ export const processRequest = (
       })
 
       stream.on('error', error => {
-        if (currentStream === stream) currentStream = null
         stream.unpipe()
         capacitor.destroy(exitError || error)
```

Comment:
This line _could_ be changed to simply use `error`. However, in node v8, the order of error propagations through the HTTP stream is different, and instead of:

```
BadRequestError: Request disconnected during file upload stream parsing.
```

we get the error created by dicer:

```
Error: Part terminated early due to unexpected end of multipart data
```

This is conceptually a "secondary" error, but is the one that propagates. For this reason, we will use `exitError` if it's set.

We could potentially remove the `|| error`, but there may be value in reporting any errors that are internal to the libraries that are used to manage these streams (busboy, dicer, node's stream lib, etc). So increasing coverage here may not be worth removing code designed to handle an "unexpected error,"  but I'll leave this to your discretion.